minio/helm/directpv/certs/ca.pem

```diff
@@ -0,0 +1,33 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
should we check this in?

Reply:
Sadly I think we have to. Their CLI generates the certificates at installation time, which were for a different namespace so I needed to generate new ones manually. I don't think we'll be able to use cert-manager since the operator is coded to look for `ca.pem`, `cert.pem` and `key.pem` and cert-manager will create `ca.crt`, `tls.crt` and `tls.key`.

If they move away from only being able to deploy directpv with their CLI that would make me very happy haha.